generator/centers_table_builder.cpp

```diff
@@ -0,0 +1,61 @@
+#include "generator/centers_table_builder.hpp"
+
+#include "search/search_trie.hpp"
+
+#include "indexer/centers_table.hpp"
+#include "indexer/feature_algo.hpp"
+#include "indexer/features_offsets_table.hpp"
+#include "indexer/features_vector.hpp"
+
+#include "coding/file_container.hpp"
+
+#include "platform/mwm_traits.hpp"
+#include "platform/platform.hpp"
+
+#include "defines.hpp"
+
+namespace indexer
+{
+bool BuildCentersTableFromDataFile(string const & filename, bool forceRebuild)
+{
+  search::CentersTableBuilder builder;
+
+  {
+    auto const & platform = GetPlatform();
+    FilesContainerR rcont(platform.GetReader(filename, "f"));
```

Comment:
Platform::GetReader(...) calls Platform::ReadPathForFile(...) and Platform::ReadPathForFile could throw MYTHROW(FileAbsentException, ... .
Could you please write try/catch in BuildCentersTableFromDataFile?